### PR TITLE
Add moFF to BioTools

### PR DESCRIPTION
**Pull request by Joott**
Add a wrapper for the docker container of moFF.

### CODE REVIEW

File: src/BioFSharp.BioTools/MoFF.fs [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+///'mass' : mass of the peptide
+///'charge' : charge of the ionized peptide.
+/// ------ Argentini et al. Nature Methods. 2016 12(13):964–966.
+
+module MoFF =
+
+    open BioContainer
+    open FSharpAux
+
+    type InputOption =
```
Comment (kMutagene) [inline src/BioFSharp.BioTools/MoFF.fs]: Add markup comment here

File: src/BioFSharp.BioTools/MoFF.fs [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+            | RawFile (pathList)                  -> List.append ["--raw_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+
+        static member make = function
+            | Ms2PeptidesInputFileFolder (path)   -> ["--loc_in" ; path]
+            | RawFilesFolder (path)               -> ["--raw_repo" ; path]
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] pathList 
+            | RawFile (pathList)                  -> List.append ["--raw_list"] pathList 
+
+    type MatchBetweenRunsOptions =
```
Comment (kMutagene) [inline src/BioFSharp.BioTools/MoFF.fs]: Add markup comment here (e.g. "Options for MatchBetweenRuns: explanation what MBR means")

File: src/BioFSharp.BioTools/MoFF.fs [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+        static member make = function
+            | FilterInputFiles (name)                 -> ["--sample" ; string name]
+            | FileExtensionsInputFile (fileExtension) -> ["--ext" ; string fileExtension]
+            | FileNameMBRLogFile (name)               -> ["--log_label" ; string name]
+            | WidthForOutlierFiltering (width)        -> ["--w_filt" ; string width]
+            | OutlierFilter (boolean)                 -> ["--out_flag" ; string boolean]
+            | WeightedRTModelCombination (boolean)    -> ["--w_comb" ; string boolean]
+
+    ///mzML must be specified using --tsv_list | --raw_list (Ms2PeptidesInputFile & RawFile)
```
Comment (kMutagene) [inline src/BioFSharp.BioTools/MoFF.fs]: Additionally to this text, add suggestions from above

File: src/BioFSharp.BioTools/MoFF.fs [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+                //TODO: check if this needs actually a number, could be that it needs nothing
+                | PeptideSummaryFile                  -> ["--peptide_summary"]
+                | PeptideSummaryFileName (name)       -> ["--tag_pepsum" ; name]
+                | OutputFileFolder (path)             -> ["--loc_out" ; path]
+                | FilterMatchedPeak (boolean)         -> ["--match_filter" ; string boolean]
+                | PtmFile (path)                      -> ["--ptm_file" ; path]
+                | FilteringThresholdQuantileValue (f) -> ["--quantile_thr_filtering" ; string f]
+                | SampleSizeForThreshold (f)          -> ["--sample_size" ; string f]
+
+    type EntireWorkflowOptions =
```
Comment (kMutagene) [inline src/BioFSharp.BioTools/MoFF.fs]: Same

File: src/BioFSharp.BioTools/MoFF.fs [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+            ///TODO: check if this needs actually a number, could be that it needs nothing
+            | PeptideSummaryFile                      -> ["--peptide_summary"]
+            | PeptideSummaryFileName (name)           -> ["--tag_pepsum" ; name]
+            | OutputFileFolder (path)                 -> ["--loc_out" ; path]
+            | FilterMatchedPeak (boolean)             -> ["--match_filter" ; string boolean]
+            | PtmFile (path)                          -> ["--ptm_file" ; path]
+            | FilteringThresholdQuantileValue (f)     -> ["--quantile_thr_filtering" ; string f]
+            | SampleSizeForThreshold (f)              -> ["--sample_size" ; string f]
+
+    type MoFFParameters =
```
Comment (kMutagene) [inline src/BioFSharp.BioTools/MoFF.fs]: And here

File: src/BioFSharp.BioTools/MoFF.fs (hunk 3 above)
Comment (Joott) [reply src/BioFSharp.BioTools/MoFF.fs]: done

File: src/BioFSharp.BioTools/MoFF.fs (hunk 2 above)
Comment (Joott) [reply src/BioFSharp.BioTools/MoFF.fs]: done

File: src/BioFSharp.BioTools/MoFF.fs (hunk 1 above)
Comment (Joott) [reply src/BioFSharp.BioTools/MoFF.fs]: added markup comment here

**Pull request merged**